packages/test-runner-intercept/browser/index.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Intercept a module in order to change its implementation.
+ * @param moduleName The name of the module to intercept. When intercepting the import of a module, the name should exactly match the name of the import.
+ * @returns Writable object with every named export of the intercepted module as a property. 
+ * If the module to be intercepted contains a default export, the export will be available in the `default` property. Only function expressions are supported when intercepting default exports.
+ */
+export function interceptModule(moduleName: string): Promise<Record<string, func> | void>;
```

Comment:
is this auto-generated?

Reply:
No, this is a browser file, so there is no src/dist for it. Also, there are no scripts for browser folders.
This is the same convention as used in `test-runner-commands`, `test-runner-core` and `test-runner-visual-regression`.